packages/api/src/router/board.ts

```diff
@@ -248,6 +248,7 @@ export const boardRouter = createTRPCRouter({
               isPublic: input.isPublic,
               columnCount: input.columnCount,
               creatorId: ctx.session.user.id,
+              itemRadius: "lg",
```

Comment:
You don't need the item radius in the insert statements as it is the default in the db